modules/json/docs/README.md

```diff
@@ -0,0 +1,47 @@
+# JSONLoader
+
+JSON is supported natively and `JSON.parse`. However there are a number of reasons why you might consider a custom JSON loader:
+
+- Error messages: `JSON.parse` tends to have unhelpful error messages
+
+## Options
+
+- `streaming`: `auto` - assume the file contains multiple top level JSON objects (or a top-level array of objects).
+- `lengthPrefixed`: `auto` - if the first value in the file is a number, assumes the file is length prefixed.
+
+## Streaming JSON
+
+### Autodetection of streaming JSON
+
+A number of hints can be used to determine if the data is formatted using a streaming JSON format
+
+- if the filename extension is `.jsonl`
+- if the MIMETYPE is `application/json-seq`
+- if the first value in the file is a number, assume the file is length prefixed.
+
+For data in non-streaming JSON format, the presence of a top-level array will start streaming of objects.
+
+For embedded arrays, a path specifier may need to be supplied (or could look for first array).
+
+### Streaming JSON Formats
```

Comment:
Does it support GeoJSON feature collection yet?

Reply:
> Does it support GeoJSON feature collection yet?

No that's an extra wrinkle that I'd like to tackle separately.

Reply:
Now supports GeoJSON (it simply "streams" the first array (i.e. non-object) it encounters in the JSON object, which would typically be the contents of the "features" field in a GeoJSON file).